examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>NITPICK: Leave one empty space before and after <code>==</code></p><p> Remove the comments that are at the bottom of the cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yeah this is the original notebook from the CDC. I could just run `black-nb` on it.